flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/job-checkpoints.component.html

```diff
@@ -550,6 +550,24 @@
               {{ checkPointConfig['checkpoints_after_tasks_finish'] ? 'Enabled' : 'Disabled' }}
             </td>
           </tr>
+          <tr
+            *ngIf="
```

Comment:
I think we can add a more line to tell whether changelog state-backend is enabled. If enabled, we print `true` with two more lines. Otherwise, we just print `false` without another two lines.

Reply:
Thanks for reviewing, and I add one `Changelog` line. Here are pictures:
![image](https://user-images.githubusercontent.com/18653940/167641222-60d9ddca-b463-4ca9-b4ce-bafc90673b5c.png)

![image](https://user-images.githubusercontent.com/18653940/167641389-6da023d8-869d-447a-af77-c121cbc1af2d.png)